auth/auth.go

```diff
@@ -58,6 +58,24 @@ type Token struct {
 	Claims   map[string]interface{} `json:"-"`
 }
 
+func (t *Token) decodeFrom(s string) error {
+	// Decode into a regular map to access custom claims.
+	claims := make(map[string]interface{})
+	if err := decode(s, &claims); err != nil {
+		return err
+	}
+	// Now decode into Token to access the standard claims.
+	if err := decode(s, t); err != nil {
+		return err
+	}
+
+	for _, r := range []string{"iss", "aud", "exp", "iat", "sub", "uid"} {
```

Comment:
Maybe another comment here like "Delete the standard claims from the custom claims map."

Reply:
Done